modules/ossm-migrating-read-me-independently-managed-gateways.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-assembly.adoc
+
+//Start of an overall Migrating section.
+//Section is most likely to be reworked/restructured with OSSM 2 to OSSM 3 migration guides for GA. Unknown how many migration guides there are at this time (11/11/2024). It would be beneficial to be able to link from differences to the relevent migration guide so that users A) understand the change, esp significant changes like new operator, installing tracing and Kiali separately, gateways, etc.
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-migrating-read-me-independently-managed-gateways_{context}"]
+= Independently managed gateways
+
+In Istio, gateways are used to manage traffic entering (ingress) and exiting (egress) the mesh. {SMProductName} 2 deployed and managed an ingress gateway and an egress gateway with the Service Mesh control plane. Both an ingress gateway and an egress gateway were configured using the `ServiceMeshControlPlane` resource.
+
+The {SMProduct} 3 Operator does not create or manage gateways.
+
+Instead, gateways in {SMProduct} 3 are created and managed independent of the operator and control plane using gateway injection or the Kubernetes Gateway API. This provides greater flexibility and ensures that gateways can be fully customized and managed as part of a Red Hat OpenShift GitOps pipeline. This allows the gateways to be deployed and managed alongside their applications with the same lifecycle.
+
+//Note to add GitOps attributes to common-attributes file. Adding attributes is being handled by a different Jira issue, and is outside the scope of this PR.
+
+This change was made for two reasons:
+
+* To start with a gateway configuration that can expand over time to meet the more robust needs of a production environment.
+* Gateways are better managed together with their corresponding workloads.
+
+Gateways may continue to be deployed onto nodes or namespaces independent of applications. For example, a centralized gateway node. Istio gateways also remain eligible to be deployed on {product-title} infrastructure nodes.
```

Comment:
{product-title} expands to "OpenShift Service Mesh", but the concept of an infrastructure node is a general OpenShift platform concept, not specific to service mesh, so it should just say "OpenShift infrastructure nodes".
See: https://docs.openshift.com/container-platform/4.17/nodes/nodes/nodes-nodes-creating-infrastructure-nodes.html